[2024-06-05T15:28:20.392+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-05T15:28:20.406+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: metrics_calculator.statistics_calculator manual__2024-06-05T15:28:18.063199+00:00 [queued]>
[2024-06-05T15:28:20.410+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: metrics_calculator.statistics_calculator manual__2024-06-05T15:28:18.063199+00:00 [queued]>
[2024-06-05T15:28:20.410+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-06-05T15:28:20.419+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): statistics_calculator> on 2024-06-05 15:28:18.063199+00:00
[2024-06-05T15:28:20.426+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=359) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-05T15:28:20.427+0000] {standard_task_runner.py:63} INFO - Started process 361 to run task
[2024-06-05T15:28:20.428+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'metrics_calculator', 'statistics_calculator', 'manual__2024-06-05T15:28:18.063199+00:00', '--job-id', '54', '--raw', '--subdir', 'DAGS_FOLDER/dag_reader.py', '--cfg-path', '/tmp/tmpnaoix6nr']
[2024-06-05T15:28:20.430+0000] {standard_task_runner.py:91} INFO - Job 54: Subtask statistics_calculator
[2024-06-05T15:28:20.528+0000] {task_command.py:426} INFO - Running <TaskInstance: metrics_calculator.statistics_calculator manual__2024-06-05T15:28:18.063199+00:00 [running]> on host 9d270a478535
[2024-06-05T15:28:20.615+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='mehdizarria' AIRFLOW_CTX_DAG_ID='metrics_calculator' AIRFLOW_CTX_TASK_ID='statistics_calculator' AIRFLOW_CTX_EXECUTION_DATE='2024-06-05T15:28:18.063199+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-05T15:28:18.063199+00:00'
[2024-06-05T15:28:20.617+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-05T15:28:23.281+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-05T15:28:23.285+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-05T15:28:23.323+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=metrics_calculator, task_id=statistics_calculator, run_id=manual__2024-06-05T15:28:18.063199+00:00, execution_date=20240605T152818, start_date=20240605T152820, end_date=20240605T152823
[2024-06-05T15:28:23.384+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-05T15:28:23.434+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-05T15:28:23.435+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
